test/3_withdrawals.js

```diff
@@ -5238,14 +5238,29 @@ contract('Cashier withdrawals ', async (addresses) => {
         );
       });
 
+      it('Disaster State should be falsy value initially', async () => {
+        const disasterState = await contractCashier.isDisasterStateSet();
+
+        assert.isFalse(disasterState);
+      });
+
       it('Admin should be able to set the Cashier at disaster state', async () => {
-        const tx = await contractCashier.setDisasterState();
+        const cashier = await Cashier.at(
+          await contractBosonRouter.getCashierAddress()
+        );
+        const tx = await cashier.setDisasterState();
 
         truffleAssert.eventEmitted(tx, 'LogDisasterStateSet', (ev) => {
           return ev._triggeredBy == users.deployer.address;
         });
       });
 
+      it('Disaster State should be set successfully', async () => {
+        const disasterState = await contractCashier.isDisasterStateSet();
```

Comment:
Each test case should be independently runnable  (i.g. with `.only`) and should not rely on state set in a previous test case. Change the test case to make it independent.